src/TableView.jl

```diff
@@ -259,6 +258,14 @@ function _showtable_async!(w, schema, names, types, rows, coldefs, tablelength,
     onimport(w, handler)
 end
 
+function _is_javascript_safe(x::Integer)
+    -Int128(2^53)-1 < x < Int128(2^53)-1
+end
+
+function _is_javascript_safe(x::AbstractFloat)
+    -2^53-1 < x < 2^53-1
```

Comment:
I think this needs to be `Int64(2)^53` because int literals on 32-bit systems are `Int32`, which overflows here (same above).

Reply:
Indeed, good catch - thanks!  Fixed in the most recent commit.